src/index.css

```diff
@@ -2,16 +2,76 @@
   font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
   line-height: 1.5;
   font-weight: 400;
-
+  
   color-scheme: light dark;
   color: rgba(255, 255, 255, 0.87);
```

Comment:
We need to avoid having fixed colors and use css variables instead, as we will need to implement light/dark mode feature. Having fixed colors will make this very difficult to implement.